src/app/app.component.html

```diff
@@ -1 +1,4 @@
-<router-outlet></router-outlet>
+<div class="sky-pages-cloak" [ngClass]="{ 'visible': isReady }">
```

Comment:
I'd rather see an `ngClass` attribute that adds a CSS class when `!isReady` so that it's removed when `isReady` is true.  That way you only need one CSS class instead of two.  As for the name, I'd use something like `skyux-app-loading`.